templates/cluster-template-eks-ipv6.yaml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
@richardcase I think I added the right e2e test config. Do I need to modify eks_test.go in any way to check for anything? :D

Also, check out e2e_eks_ipv6_conf.yaml. :))

Reply:
We probably need to either:

- Add a new test case to `eks_test.go`
- Add a new test file (i.e. `eks_ipv6_test.go`)

Either way, we would write a new test  that uses a IPV6 template. Lets chat via slack/zoom